### PR TITLE
Update README for Bandits

### PR DESCRIPTION
**Pull request by typotter**
- Update the README file to include instructions for bandits
  - Also update example code to use the new `LoggerInterface`
  - Also also add handy `toArray` method to assignment log event
- Lint the codebase to catch formatting missed in last PR

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 [Eppo](https://www.geteppo.com/) is a modular flagging and experimentation analysis tool. Eppo's PHP SDK is built to make assignments in multi-user server side contexts, compatible with PHP 7.3 and above. Before proceeding you'll need an Eppo account.
 
 ## Features
-
+- Bandits
```
Comment (aarsilv) [inline README.md:6]: Consider listing this last as most customers will be more interested in the other features

File: README.md [hunk 2]
```diff
@@ -52,6 +52,44 @@ if ($assignment !== 'defaultValue') {
 }
 ```
 
+### Select a Bandit Action
+This SDK supports [Multi-armed Contextual Bandits](https://docs.geteppo.com/contextual-bandits/).
```
Comment (aarsilv) [inline README.md:56]: ❤️ Love the documentation link!

File: README.md [hunk 3]
```diff
@@ -52,6 +52,44 @@ if ($assignment !== 'defaultValue') {
...
+
+```php
+$subjectContext = [
+    'age' => 30, // Gets interpreted as a Numeric Attribute
+    'country' => 'uk', // Categorical Attribute
+    'pricingTier' => '1'  // NOTE: Deliberately setting to string causes this to be treated as a Categorical Attribute
```
Comment (aarsilv) [inline README.md:62]: Excellent example

File: README.md [hunk 4]
```diff
@@ -52,6 +52,44 @@ if ($assignment !== 'defaultValue') {
...
+];
+
+$actionContexts = [
+    'nike' => [
+        'brandLoyalty' => 0.4,
+        'from' => 'usa'
+    ],
+    'adidas' => [
+
```
Comment (aarsilv) [inline README.md:71]: extraneous line?

**Review by aarsilv — APPROVED** [review]
Looking good! Do we need to bump a version somewhere for composer or is that handled outside of the code base?

**Comment by typotter** [comment]
Thanks @aarsilv 
Composer/Packagist automatically watches the repo for a published release. It will take the version string from the tag of whatever release it parses, so we don't need to maintain a version string in the codebase (in fact, composer.json will fail validation if we hardcode a version).

**Pull request merged**